.github/workflows/dockerimage.yml

```diff
@@ -25,5 +28,5 @@ jobs:
            --tag owasp/modsecurity:$(./version.sh ${{ matrix.image }} -vvv)
 
     - name: Push ${{ matrix.image }}
-      if: github.ref == 'refs/heads/master'
+      if: github.event_name == 'push' && github.ref == 'refs/heads/master'
```

Comment:
Same here. Note that the use of `if` (in our implementation here) is really just a workaround; it's procedural, but we should really favor a declarative implementation style (that's what `on: push` is about).

Reply:
You are exactly right! (bad copy and paste)